src/DropdownToggle.js

```diff
@@ -31,11 +40,13 @@ export default class DropdownToggle extends React.Component {
 DropdownToggle.defaultProps = {
   open: false,
   useAnchor: false,
-  bsRole: 'toggle'
+  bsRole: 'toggle',
+  bsClass: 'dropdown'
```

Comment:
for consistency this should be defined via the bsClass helper in bootstrapUtils, other than that looks good :thumbsup: